### ISSUE TITLE
Provide a method to retrieve parameters in the API hooks

### ISSUE THREAD

**Issue by shilman** [storybookjs/test-runner#97]
Currently there is no stylized way to retrieve story parameters in API hooks.

This would be useful for reusing:
- viewports configurations
- a11y parameters
- etc.

This could be provided automatically by the test-runner as part of the `context` argument.

```js
  async preRender(page, context) {
    const { parameters } = context;
  },
```

Or maybe on=demand:

```js
  async preRender(page, context) {
    const parameters = await getParameters(page, context);
    // OR
    const parameters = await context.getParameters(page);
  },
```

The benefit of on-demand is that the user would only "pay" the performance cost of retrieving parameters when they are actually used.

**Comment by lacolaco** [comment]
I strongly want parameters for viewports configurations and story skipping!

**Comment by yannbf** [comment]
Hey peeps! This was released in `v0.2.0`, please check the details [here](https://github.com/storybookjs/test-runner/pull/125). Thanks for using this library!
